src/main/kotlin/org/rust/cargo/runconfig/ui/RsCommandConfigurationEditor.kt

```diff
@@ -34,6 +34,21 @@ abstract class RsCommandConfigurationEditor<T : LocatableConfigurationBase<RunPr
 
     protected val workingDirectory: LabeledComponent<TextFieldWithBrowseButton> =
         WorkingDirectoryComponent()
+
+    protected val programInputRedirect: ProgramInputRedirectPanel =
+        ProgramInputRedirectPanel()
+
+    override fun resetEditorFrom(configuration: T) {
+        command.text = configuration.command
+        workingDirectory.component.text = configuration.workingDirectory?.toString().orEmpty()
+        programInputRedirect.reset(configuration)
+    }
+
+    override fun applyEditorTo(configuration: T) {
+        configuration.command = command.text
+        configuration.workingDirectory = currentWorkingDirectory
+        programInputRedirect.applyTo(configuration)
+    }
```

Comment:
Also, let's move all stuff related to `programInputRedirect` to `CargoCommandConfigurationEditor` since we don't need it in `WasmPackCommandConfigurationEditor`